andela/ah-lobos-frontend
# #166789827 Setup react router

#### What does this PR do?
This pull request sets up the project to use react router
#### Description of Task to be completed?
We are going to use react router for our client. Install and configure the router creating a dummy route for login and the "/" page to test the configuration.
Create a components and views file structure where view pages are in the views folder but individual components are in the components folder.
#### How should this be manually tested?
> clone the repository 
```
git clone https://github.com/andela/ah-lobos-frontend.git
```
> navigate into the project directory
```
cd ah-lobos-frontend
```
> install all dependencies
```
npm install
```
> start application
```
npm start
```
> your browser should automatically open with a similar interface as the image bellow
<img width="1423" alt="Screen Shot 2019-08-14 at 17 54 58" src="https://user-images.githubusercontent.com/30442301/63036091-af8ea100-bebc-11e9-89cd-3c6221ca41d7.png">

> Click on the login button on the top right corner of your screen and it will redirect you to a login page bellow
<img width="1423" alt="Screen Shot 2019-08-14 at 17 57 22" src="https://user-images.githubusercontent.com/30442301/63036260-fed4d180-bebc-11e9-8cd0-d22460728fbf.png">

#### Any background context you want to provide
- N/A

#### What are the relevant pivotal tracker stories
[#166789827](https://www.pivotaltracker.com/story/show/166789827)

#### Questions?
- N/A
